lib/types/rules/ecmascript-6.d.ts

```diff
@@ -27,6 +27,37 @@
 
 import { Linter } from "../index";
 
+interface NoRestrictedImportPathCommonOptions {
```

Comment:
Sorry, I think I missed this before - I think it'd be good to `export` all the types in this file. That way they're easier for consumers to `import` ad hoc.

```suggestion
export interface NoRestrictedImportPathCommonOptions {
```

https://github.com/typescript-eslint/typescript-eslint/issues/7670 has points on more.

Reply:
Exported all types 👍🏻